### ISSUE TITLE
App crashes when casting

### ISSUE THREAD

**Issue by otakupasi** [proxer/ProxerAndroid#26]
As soon as I click the Cast Button and select the Cast device (Chromecast 2nd Generation) the following Exception is Thrown:

<details><summary>Exception</summary><p>  

   ```  
Stack trace:  
java.lang.NullPointerException: Attempt to write to field 'boolean h5.i.a.d.i.c.u7.a' on a null object reference
	at h5.i.a.d.i.c.h9.a(com.google.android.gms:play-services-cast@@17.1.0:3)
	at h5.i.a.d.i.c.w9.a(com.google.android.gms:play-services-cast@@17.1.0:603)
	at h5.i.a.d.i.c.m8.c(com.google.android.gms:play-services-cast@@17.1.0:5)
	at h5.i.a.d.i.c.m8.d(com.google.android.gms:play-services-cast@@17.1.0:1)
	at h5.i.a.d.i.c.x.a(com.google.android.gms:play-services-cast-framework@@17.1.0:1)
	at h5.i.a.d.i.c.v.a(com.google.android.gms:play-services-cast-framework@@17.1.0:56)
	at h5.i.a.d.d.y.h0.a(com.google.android.gms:play-services-cast-framework@@17.1.0:62)
	at h5.i.a.d.i.c.b.onTransact(com.google.android.gms:play-services-cast@@17.1.0:3)
	at android.os.Binder.transact(Binder.java:914)
	at jw.b(:com.google.android.gms.dynamite_dynamitemodulesc@19629081@19.6.29 (120400-0):14)
	at vw.a(:com.google.android.gms.dynamite_dynamitemodulesc@19629081@19.6.29 (120400-0):39)
	at wi.b(:com.google.android.gms.dynamite_dynamitemodulesc@19629081@19.6.29 (120400-0):50)
	at wj.a(:com.google.android.gms.dynamite_dynamitemodulesc@19629081@19.6.29 (120400-0):42)
	at wu.c(:com.google.android.gms.dynamite_dynamitemodulesc@19629081@19.6.29 (120400-0):21)
	at ws.a(:com.google.android.gms.dynamite_dynamitemodulesc@19629081@19.6.29 (120400-0):11)
	at jx.onTransact(:com.google.android.gms.dynamite_dynamitemodulesc@19629081@19.6.29 (120400-0):5)
	at android.os.Binder.transact(Binder.java:914)
	at h5.i.a.d.i.c.u.b(com.google.android.gms:play-services-cast@@17.1.0:2)
	at h5.i.a.d.i.c.l.d(com.google.android.gms:play-services-cast-framework@@17.1.0:7)
	at f5.v.l.o.a(MediaRouter.java:12)
	at f5.v.l.o.handleMessage(MediaRouter.java:20)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7356)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)  
```  
</p></details>

App-Version: 1.9.4
Device(Smartphone):
Google Pixel 3a
Running on Android 10 
Not-Rooted
Android Build: QP1A.191105.003

Cast-Device(Chromecast):
Google Chromecast Second Generation
Firmware-Version: 1.36.159268

Wifi-Information:
Download-Speed: 12.4
Upload-Speed: 8.89
Latency: 25ms
Number of Devices: 16

**Comment by rubengees** [comment]
Hi, thanks for the report! This is known, but I have not yet found the cause.
Will update here and publish a new version once found and fixed :)

**Comment by rubengees** [comment]
This is now fixed (9354af7f5d252fa2b1f69d3b3824a67ba6407f6a) and an update will come soon.
